test/cosmoz-bottom-bar-view.html

```diff
@@ -98,10 +98,65 @@
 					assert.isFalse(bottomBar.visible);
 					done();
 				}, 50);
+			});
+
+			test('bottomBar is visible when scrolling up', function (done) {
```

Comment:
The tests can use arrow function syntax.

Reply:
Replaced tests with arrow function syntax. Also did it in `cosmoz-bottom-bar` tests.